packages/okta-react/README.md

```diff
@@ -352,76 +450,165 @@ class App extends Component {
+
+#### `authService.getUser()`
 
 Returns the result of the OpenID Connect `/userinfo` endpoint if an access token exists.
 
-#### `auth.getIdToken()`
+#### `authService.getIdToken()`
 
-Retrieves the id token from storage if it exists.
+Retrieves the id token from storage if it exists.  Devs should prefer to consult the synchronous results emitted from subscribing to the `authStateChange` event using `authService.on()`
 
-#### `auth.getAccessToken()`
+#### `authService.getAccessToken()`
 
-Retrieves the access token from storage if it exists.
+Retrieves the access token from storage if it exists.  Devs should prefer to consult the synchronous results emitted from subscribing to the `authStateChange` event.
 
-#### `auth.login(fromUri, additionalParams)`
+#### `authService.login(fromUri, additionalParams)`
 
-Calls `onAuthRequired` or redirects to Okta if `onAuthRequired` is undefined. This method accepts a `fromUri` parameter to push the user to after successful authentication, and an optional `additionalParams` object.
+Calls the optional `onAuthRequired` passed to `<Security>` or redirects to Okta if `onAuthRequired` is undefined. This method accepts a `fromUri` parameter to push the user to after successful authentication, and an optional `additionalParams` object.
 
 For more information on `additionalParams`, see the [`auth.redirect`](#authredirectadditionalparams) method below.
 
-#### `auth.logout(uri)`
+#### `authService.logout(uri)`
 
 Terminates the user's session in Okta and clears all stored tokens. Accepts an optional `uri` parameter to push the user to after logout.
 
-#### `auth.redirect(additionalParams)`
+#### `authService.redirect(additionalParams)`
 
 Performs a full-page redirect to Okta with optional request parameters.
 
 The `additionalParams` are mapped to Okta's [`/authorize` request parameters](https://developer.okta.com/docs/api/resources/oidc#authorize). This will override any existing [configuration](#configuration-options). As an example, if you have an Okta `sessionToken`, you can bypass the full-page redirect by passing in this token. This is recommended when using the [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget). Simply pass in a `sessionToken` into the `redirect` method as follows:
 
 ```jsx
-auth.redirect({
+authService.redirect({
   sessionToken: '{sampleSessionToken}'
 });
 ```
 
 > Note: For information on obtaining a `sessionToken` using the [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget), please see the [`renderEl()` example](https://github.com/okta/okta-signin-widget#rendereloptions-success-error).
 
-#### `auth.handleAuthentication()`
+#### `authService.handleAuthentication()`
+
+Parses tokens from the url and stores them.  Done when handling a redirect back to the application from the issuer.  See the LoginCallback component for an example of use.
+
+### `authService.setFromUri(uri, queryParams)`
```

Comment:
This method does not take `queryParams` anymore. It takes a URI string. If a relative path is passed, it will be converted to an absolute URI before it is stored.